packages/components/src/chart/d3chart/legend.js

```diff
@@ -158,6 +163,7 @@ D3Legend.propTypes = {
 };
 
 D3Legend.defaultProps = {
+	interactive: true,
```

Comment:
We will want to add some documentation to the CHANGELOG for the components module to reflect the updates.